ui/src/home/index.jsx

```diff
@@ -1,48 +1,52 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Row, Col } from 'antd';
 
 import SearchBoxContainer from '../common/containers/SearchBoxContainer';
 import HowToSearch from './components/HowToSearch';
 import './index.scss';
 import DocumentHead from '../common/components/DocumentHead';
+import HomePageSection from './components/HomePageSection';
+import HowToSubmit from './components/HowToSubmit';
+import CollaborationLogos from './components/CollaborationLogos';
 
+const INSPIRE_DESCRIPTION =
```

Comment:
can we read them from the config instead and default to this?

Reply:
maybe when we plan to configure it, we can do that.